packages/Button/src/Button.styles.js

```diff
@@ -182,13 +182,14 @@ export const sizeStyles = {
 export const kindStyles = ({ isDisabled }) => ({
   [types.DEFAULT]: css`
     ${skeuomorphicStyles}
-
     background-color: ${tokens.color.white};
     background-image: linear-gradient(${tokens.color.blackLighten90}, ${tokens.color.blackLighten70});
     color: ${tokens.color.black};
 
-    &:hover {
+    &:hover, &:visited {
```

Comment:
I don't think you want `visited` styles to have the grey background... that one should be _only_ for `hover`.

`background-color`, `background-image`, `color`, and `text-decoration` could all be applied once in a block for `&, &:hover, &:visited, &:active` as long as there was another `&:hover` block _after_ that with a `background` override.